homeworkEight-MaterialUI/materialUI/src/app/users/users.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from '@angular/core';
+import { PeopleService} from '../people.service';
+import { Observable } from 'rxjs';
+import { PhotoserviceService} from '../photoservice.service';
+import { UserphotosService } from './userphotos.service';
+
+@Component({
+  selector: 'app-users',
+  templateUrl: './users.component.html',
+  styleUrls: ['./users.component.css']
+})
+export class UsersComponent implements OnInit {
+
+  public people$:Observable<any>;
+  public photos$:Observable<any>;
+
+  constructor( private peopleService: PeopleService, private photosService: PhotoserviceService, public usersphotosService: UserphotosService ) { }
+
+  
+
+  ngOnInit() {
+    this.people$ =  this.peopleService.fetchPeople();
+    this.photos$ =  this.photosService.fetchPhotos();
+    this.usersphotosService.getAllData().subscribe( data => {
+      console.log(data);
+      const users = data[0];
+      const photos = data[1];
+
+      const dataUser = users.map(user => {
```

Comment:
adauga in constructor dataUser: []
in loc de const dataUser =   pune this.dataUser -> in html nu stie sa iti citeasca const
 in html sterge te rog async pentru ca aici nu si un observable -> observable este doar in primul caz this.people$ =  this.peopleService.fetchPeople();

Reply:
Okay, am facut asta dar nu mi se pare ca s-a schimbat mai nimic :(. Cred ca problema o am la iteratul prin acest array. 
![image](https://user-images.githubusercontent.com/47689379/76007377-9f9ce400-5f16-11ea-8bb3-f46ae47ea981.png)
Din array-ul dataUser pentru fiecare iteratie vreau sa accesez primul array(user) care la randul lui are proprietatea la care vreau sa ajung -> user.name. Deci am nevoie de primul index din fiecare obiect de pe array-ul mare.
Ma omoara umpic.

Reply:
pai faci iteratie dupa user si trebuie pus in html de ex user.user.name sau user.photos.name